src/app/modules/round/round.controller.js

```diff
@@ -312,7 +312,8 @@ angular
           PlayerStats.updateBestRound(vm.player, playerScore);
         }
         var nextRound = currentRoundPlayedID.length < SelectedPlayers.getAll().length ? vm.round : vm.round + 1;
-        $location.path('summary/round/' + nextRound + '/player/' + Match.getNextPlayer(nextRound).id);
+        Match.setCurrentplayer(Match.getNextPlayer(nextRound));
```

Comment:
Se il set lo fai qui non ha senso quel che hai fatto sopra

Reply:
Ti riferisci al set in players controller?

Reply:
si